pm/stub.go

```diff
@@ -283,6 +283,7 @@ type stubSenderMonitor struct {
 	acceptable  bool
 	addFloatErr error
 	maxFloatErr error
+	isLocked    error
```

Comment:
Would be nice to follow the existing naming convention for error type fields - at the moment, this name would be better suited for a bool type.